app/src/main/java/com/devshish/internship/ui/library/LikedSongsFragment.kt

```diff
@@ -0,0 +1,36 @@
+package com.devshish.internship.ui.library
+
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.devshish.internship.R
+import com.devshish.internship.adapter.ItemSongAdapter
+import com.devshish.internship.databinding.FragmentLikedSongsBinding
+
+class LikedSongsFragment : Fragment(R.layout.fragment_liked_songs) {
+
+//    private lateinit var viewModel: LibraryViewModel
+    private val viewModel: LibraryViewModel by viewModels()
+
+    private val itemSongAdapter = ItemSongAdapter {
+        Toast.makeText(requireContext(),
```

Comment:
add viewModel.songClicked(song) method - and log into Log.d inside of this method